pgsync/sync.py

```diff
@@ -1421,8 +1433,11 @@ def _status(self, label: str) -> None:
             f"Xlog: [{self.count['xlog']:,}] => "
             f"Db: [{self.count['db']:,}] => "
             f"Redis: [{self.redis.qsize:,}] => "
-            f"{self.search_client.name}: [{self.search_client.doc_count:,}]"
-            f"...\n"
+            f"{self.search_client.name}: [{self.search_client.doc_count:,}] => "
+            f"skip-Xlog: [{self.count['skip_xlog']:,}] => "
+            f"skip-Redis: [{self.count['skip_redis']:,}] => "
+            f"Events: [{self.count['notifications']:,}]"
+            f"\n"
```

Comment:
logs as:
```
Xlog: [3,185] => Db: [72,251] => Redis: [1] => OpenSearch: [63,190] => skip-Xlog: [2,867] => skip-Redis: [9,224] => Events: [81,475]
```

let's do these:
```
tx-log [3,185] => db-changes [72,251] => queued [1] => elastic [63,190]  |||  skipped: tx-log [2,867], queue [9,224]
```

Reply:
let's leave this as is, or we i should refactor vector and grafana dashboards

Reply:
k, let's do it in the follow up PRs